toast-tk-runtime/src/main/java/io/toast/tk/runtime/report/MailReportSender.java

```diff
@@ -133,6 +116,42 @@ private String getMailBody(ICampaign campaign) {
 		return sb.toString();
 	}
 
+
+	private String getMailBody(List<ITestPage> testPages, ResultObject res) {
+		StrBuilder sb = new StrBuilder();
+
+		sb.append("Test report for ").append(testPages.size()).append(" the test pages.");
+		sb.appendNewLine();
+
+		// First test start time
+		if (CollectionUtils.isNotEmpty(testPages)) {
+			long startDateMillis = testPages.get(0).getStartDateTime();
+			LocalDateTime startDateTime =
+					LocalDateTime.ofInstant(Instant.ofEpochMilli(startDateMillis), ZoneId.systemDefault());
+			sb.append("Start time: ").append(getDateAsString(startDateTime)).appendNewLine();
+		}
+		sb.appendNewLine();
+
+		// Success number
+		sb.append(res.getTotalSuccess()).append(" tests in success").appendNewLine();
+
+		// Failure number
+		long failureNumber = res.getTotalErrors() + res.getTotalTechnical();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this addition operation to a "long". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)